lib/cPanel/TaskQueue.pm

```diff
@@ -859,13 +859,15 @@ END { undef %valid_processors }    # case CPANEL-10871 to avoid a SEGV during gl
             }
             else {
 
+                $self->_process_overrides($task);
+                next if $self->_is_duplicate_command($task);
+
                 # move 'no longer deferred' tasks in reverse order to processing list
-                unshift @proc, $task;
+                unshift @{ $self->{queue_waiting} }, $task;
             }
         }
 
         # update queues
-        $self->{queue_waiting} = [ @proc, @{ $self->{queue_waiting} } ] if @proc;
```

Comment:
Since `@proc` is no longer used, please remove its declaration on line 855.

Reply:
Good point.  I'll do that.